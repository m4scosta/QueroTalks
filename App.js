import { StackNavigator } from 'react-navigation';
import {
  Slide1,
  Slide2,
  Slide3,
  Slide4,
  Slide5,
  Slide6,
  Slide7,
  Slide8,
} from './js/Slides';

export default App = StackNavigator({
    Slide1: { screen: Slide1 },
    Slide2: { screen: Slide2 },
    Slide3: { screen: Slide3 },
    Slide4: { screen: Slide4 },
    Slide5: { screen: Slide5 },
    Slide6: { screen: Slide6 },
    Slide7: { screen: Slide7 },
    Slide8: { screen: Slide8 },
  }, {
    headerMode: 'none'
  }
);
