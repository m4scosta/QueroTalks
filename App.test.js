import React from 'react';
import App from './App';

import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const rendered = renderer.create(<App />).toJSON();
  expect(rendered).toBeTruthy();
});


it('renders the correct content', () => {
  const slides = renderer.create(<App />).toJSON();
  expect(slides).toMatchSnapshot();
});
