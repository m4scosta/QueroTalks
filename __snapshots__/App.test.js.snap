// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the correct content 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    onMoveShouldSetResponder={[Function]}
    onMoveShouldSetResponderCapture={[Function]}
    onResponderEnd={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderReject={[Function]}
    onResponderRelease={[Function]}
    onResponderStart={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onStartShouldSetResponderCapture={[Function]}
    style={
      Object {
        "flex": 1,
        "flexDirection": "column-reverse",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        collapsable={undefined}
        pointerEvents="auto"
        style={
          Object {
            "backgroundColor": "#E9E9EF",
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "shadowColor": "black",
            "shadowOffset": Object {
              "height": 0,
              "width": 0,
            },
            "shadowOpacity": 0.4,
            "shadowRadius": 10,
            "top": 0,
            "transform": Array [
              Object {
                "translateX": 0,
              },
              Object {
                "translateY": 0,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#44a0dd",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "white",
                "fontSize": 40,
                "fontWeight": "bold",
                "marginBottom": 30,
              }
            }
          >
            react-native
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "white",
                "fontSize": 20,
              }
            }
          >
            by Marcos Costa
          </Text>
          <View
            style={
              Object {
                "bottom": 0,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "left": 20,
                "position": "absolute",
                "right": 20,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "white",
                    "fontSize": 50,
                    "fontWeight": "bold",
                  },
                  Object {
                    "color": "lightgray",
                  },
                ]
              }
            >
              &lt;
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              onPress={[Function]}
              style={
                Object {
                  "color": "white",
                  "fontSize": 50,
                  "fontWeight": "bold",
                }
              }
            >
              &gt;
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
